frontend/src/libs/components/map/map.tsx

```diff
@@ -46,8 +48,12 @@ const Map: React.FC<Properties> = ({
 
         void mapService.current.calculateRouteAndTime(center, destination);
       }
+
+      for (const marker of markers ?? []) {
+        mapService.current.addMarker(marker, true);
+      }
```

Comment:
maybe let's better to add check
if(markers.length){}

Reply:
Eslint wouldn't let me do that, did as he said.